src/main/java/fr/esgi/al5/tayarim/services/ReservationService.java

```diff
@@ -424,6 +431,21 @@
 
   }
 
+  @Transactional
+  @Scheduled(cron = "0 59 23 * * *")
+  protected void doCheckStatus(){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.